mon-pix/tests/acceptance/campaigns/campaign-landing-page_test.js

```diff
@@ -0,0 +1,91 @@
+  setupApplicationTest(hooks);
+  setupMirage(hooks);
+  setupIntl(hooks, ['fr', 'en']);
+
+  let campaign;
+
+  hooks.beforeEach(function () {
+    campaign = server.create('campaign');
+  });
+
+  module('on international domain (.org)', function () {
+    module('when connected', function () {
+      module('when accessing the campaign landing page with "Français" as default language', function () {
+        test('does not display the language switcher', async function (assert) {
+          // given
+          const user = server.create('user', 'withEmail');
+
+          // when
+          await authenticate(user);
+          const screen = await visit(`/campagnes/${campaign.code}`);
+
+          // then
+          assert.strictEqual(currentURL(), `/campagnes/${campaign.code}/presentation`);
+          assert.dom(screen.getByRole('button', { name: 'Je commence' })).exists();
+          assert.dom(screen.queryByRole('button', { name: 'Français' })).doesNotExist();
+        });
+      });
+    });
+
+    module('when not connected', function () {
+      module('when accessing the fill in campaign code page with "Français" as default language', function () {
+        test('displays the fill in campaign code page with "Français" as selected language', async function (assert) {
+          // given & when
+          const screen = await visit(`/campagnes/${campaign.code}`);
+
+          // then
+          assert.strictEqual(currentURL(), `/campagnes/${campaign.code}/presentation`);
+          assert.dom(screen.getByRole('button', { name: 'Je commence' })).exists();
+        });
+
+        module('when the user select "English" as his language', function () {
+          test('displays the fill in campaign code page with "English" as selected language', async function (assert) {
+            // given & when
+            const screen = await visit(`/campagnes/${campaign.code}`);
+            await click(screen.getByRole('button', { name: 'Français' }));
+            await screen.findByRole('listbox');
+            await click(screen.getByRole('option', { name: 'English' }));
+
+            // then
+            assert.strictEqual(currentURL(), `/campagnes/${campaign.code}/presentation`);
+            assert.dom(screen.getByRole('button', { name: 'Begin' })).exists();
+          });
+        });
+      });
+
+      module('when accessing the fill in campaign code page with "English" as selected language', function () {
+        test('displays the fill in campaign code page with "English"', async function (assert) {
+          // given && when
+          const screen = await visit(`/campagnes/${campaign.code}?lang=en`);
+
+          // then
+          assert.strictEqual(currentURL(), `/campagnes/${campaign.code}/presentation`);
+          assert.dom(screen.getByRole('button', { name: 'Begin' })).exists();
+        });
+
+        module('when the user select "Français" as his language', function () {
```

Comment:
Le texte `as his language` laisse penser qu'il s'agit de la langue de l'utilisateur en BDD, or ici c'est uniquement le language du language switcher, aussi je suggère la modification suivante pour le titre du test :

```suggestion
        module('when the user select "Français" language', function () {
```